GoogleDataLogger/generate_project.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
Put or point to pod generate instruction in the README - as will as a brief summary of what the project is and how to develop it.